src/components/Header.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { View, StyleSheet, Text, TouchableOpacity } from 'react-native';
+
+import { colors } from '../constants/colors';
+import { symbols } from '../constants/symbols';
+
+export const Header = ({ title, onPressBack, onPressStar, isFavorite }) => {
+  const starStyle = isFavorite
+    ? { fontSize: 40, color: colors.GOLD }
+    : { fontSize: 40, color: colors.DARK_PURPLE };
+
+  return (
+    <View style={styles.container}>
+      <View>
+        {onPressBack && (
+          <TouchableOpacity onPress={onPressBack}>
+            <Text style={styles.backButton}>{symbols.LEFT_ANGULAR_BRACKET}</Text>
+          </TouchableOpacity>
+        )}
+      </View>
+      <Text style={styles.title} numberOfLines={1} ellipsizeMode={'tail'}>
+        {title}
+      </Text>
+      <View>
+        {onPressStar && (
+          <TouchableOpacity onPress={onPressStar}>
+            <Text style={starStyle}>{symbols.STAR}</Text>
```

Comment:
`styles.star` is not used in the file. It's better to have something like that: `style={[styles.star, isFavorite && color: colors.GOLD]} `